ci/linux/update_dependencies.sh

```diff
@@ -1,5 +1,9 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
-. .venv/bin/activate
-pip-compile --output-file=requirements.txt pyproject.toml
-pip-compile --extra=dev --output-file=dev-requirements.txt -c requirements.txt pyproject.toml
+if [[ "$OSTYPE" != "win32" && "$OSTYPE" != "msys" ]]; then
```

Comment:
This allows the CI scripts to be used under WSL.

Reply:
So the environment isn't activated on WSL?  then that is up to the user to do before running the ci scripts.... 

Also,  OSTYPE isn't set in my WSL2 Ubuntu 24.04 environment?

Reply:
Mark has been using these changes under WSL on the cloud repo's for the past year with success. Regardless, the bat scripts remain available for Windows of course.